scripts/load-images-to-quay.sh

```diff
@@ -0,0 +1,17 @@
+rm ../build/*.oci
+rm ../build-nodejs-16/*.oci
+rm ../build-nodejs-18/*.oci
+bash ./create.sh
+bash ./create-nodejs-16.sh
+bash ./create-nodejs-18.sh
+
+skopeo copy --dest-tls-verify=false oci-archive:../build/run.oci docker-daemon:quay.io/midawson/ubi8-paketo-run:latest
```

Comment:
I'm not entirely familiar with what this script is doing, but eventually I'd imagine we'd handle the publishing of the stack images via the [push image workflow](https://github.com/paketo-buildpacks/github-config/blob/main/stack/.github/workflows/push-image.yml) which we will add here eventually.  It gets triggered when we create a release. In this repository we'd set `DOCKERHUB_ORG` to paketocommunity instead, however. We may need to also make some other small edits to the workflow to make it work for this use case

Reply:
That workflow is how we manage publishing our other stacks in Paketo

Reply:
You are correct that those scripts were just my temporary way of publishing until we get the release workflow going.